osu.Game/Screens/IOsuScreen.cs

```diff
@@ -67,7 +67,13 @@ public interface IOsuScreen : IScreen
         /// Whether mod track adjustments should be applied on entering this screen.
         /// A <see langword="null"/> value means that the parent screen's value of this setting will be used.
         /// </summary>
-        bool? AllowTrackAdjustments { get; }
+        bool? ApplyModTrackAdjustments { get; }
+
+        /// <summary>
+        /// Whether control of the global track should be allowed via the music controller / now playing overlay.
+        /// A <see langword="null"/> value means that the parent screen's value of this setting will be used.
+        /// </summary>
+        bool? AllowGlobalTrackControl { get; }
```

Comment:
I guess one pain point here is that we have two levels of "global track disable" now: one being global `Beatmap.Value.Disabled` which prevents changing track, and this which prevents pausing/resuming the current track. Not sure that's necessarily bad, maybe worth an xmldoc mention.

Reply:
Yeah, this one is over-arching (also disabled changing track) and should be used in most cases I guess.

The beatmap disable one seems fine to leave around for existing usages.